cmd/test/test.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2022 Red Hat, Inc.
```

Comment:
Update year?

Reply:
https://github.com/enterprise-contract/ec-cli/pull/840

Did I overdo it?